web/compatibility.js

```diff
@@ -577,3 +577,20 @@ if (typeof PDFJS === 'undefined') {
     PDFJS.disableFullscreen = true;
   }
 })();
+
+// Provides document.currentScript support
+// Support: IE, Chrome<29.
+(function checkCurrentScript() {
+  if ('currentScript' in document) {
+    return;
+  }
+  Object.defineProperty(document, 'currentScript', {
+    get: function () {
+      var scriptTagContainer = document.body ||
```

Comment:
This method is flawed as it might not give a `<script>` tag (for async scripts). Try the following instead:

```js
var scripts = document.getElementsByTagName('script');
return scripts[scripts.length - 1];
```